src/test/scala/org/zalando/nakadi/client/KlientSpec.scala

```diff
@@ -9,53 +9,24 @@ import com.fasterxml.jackson.module.scala.DefaultScalaModule
 import com.google.common.collect.Iterators
 import com.typesafe.scalalogging.LazyLogging
 import io.undertow.util.{HeaderValues, HttpString, Headers}
-import org.scalatest.{BeforeAndAfterEach, Matchers, WordSpec}
+import org.scalatest._
+import org.scalatest.concurrent.PatienceConfiguration.{Timeout => ScalaTestTimeout}
+import org.scalatest.concurrent.ScalaFutures
 import org.zalando.nakadi.client.actor.PartitionReceiver
 import org.zalando.nakadi.client.utils.NakadiTestService
 import org.zalando.nakadi.client.utils.NakadiTestService.Builder
+import org.zalando.nakadi.client.utils.{Request => NakadiRequest}
```

Comment:
I prefer a bit longer aliases to simply `Request`, when the domain does have other possible alternatives for it (like HTTP Request).